### DISCUSSION TITLE
为什么说共产党的政治生命属性不支持对外扩张，只能对内压榨剥削

### DISCUSSION THREAD

**Discussion by Kasianian** [WeNeedHome/SummaryOfLoanSuspension#1163] [General]
一说到这个肯定有苏联粉红余孽跳出来说你是1450和行走的50w，但是作为正统中国人，就是要有一份光发一份热，不必等候矩火。

1，红色极权的发家史无非就是利用苏联共产主义传销思想，蛊惑无数农民为不可能实现的目标充当燃料并燃烧自身生命，推翻了所谓旧社会，迎来新的一轮更甚的地狱模式，其社会制度与封建制度并无差异，俗称资本主义中最低端的官僚资本主义。

2，红色极权手上沾满了无数无辜百姓被迫害的鲜血，本不应该发生的国共内战、朝鲜战争（造成朝鲜人民彻底生活在奴隶社会的悲剧）、三年饥荒、大跃进、毛泽东的斯大林模式的大清洗运动、文革被迫害致死的上千万知识分子、计划生育阉割屠杀上亿汉族婴儿，以及1989年天安门广场武力镇压屠杀学运学生等等，目前政权仍然不公开自省甚至对人民封口管制。

3，自计划经济严重限制社会生产力发展，为延续政治生命被迫推行改革开放，其四十年发展成果无非是建立在燃烧无数廉价劳动力基础上而发展出的畸形经济体制，国有垄断生产资料企业=红色权贵的官僚门阀企业、共产党垄断土地造成土地归属权混乱，有进一步被共产党红色干部进行土地兼并的情况，独裁体制下无限扩大的权力无法被有效监督和制衡，无法根治体制内生性的大范围红色权贵贪腐问题。

4，支持极权续命的税收制度已无法完成闭环，1994年的分税制改革，中央财政直接对地方财政抽成25%，造成地方财政严重依赖卖地收入，造成经济发展严重依赖房地产，每年卖地收入被用来红色权贵裙带关系户挥霍，进行的所谓大基建以供其吸血，目前大部分大基建项目都是政府的资产负债项，其中港珠澳大桥就是其缩影，往後地方政府只能靠发债维持庞大的党政体制度日，没有美帝的实力却裹上了美帝的赤字，直至土崩瓦解。

5，疫情的禁足防疫已无法使经济进行有效内循环和制造业出口，外资看空禁足防疫，大范围企业倒闭，大规模失业潮来临，同时一带一路还给外国大爷撒钱，援非项目不但撒钱还提供免费劳动力。

6，台湾军事武装是中华民族内唯一制衡红色极权的有效力量，如果共产党胆敢武力夺取台湾，以独裁吞并民主的社会，是人类文明的倒退，共产党惧怕的杀戮形象会再次曝光在全世界的注目下，国际社会集体军事和经济制裁中国，加速禁锢并缩短独裁政权政治生命，即使共产党赢得吞并台湾的战争，也无法有效对台湾人民进行长期有效的统治，属于民主社会和红色极权的互斥现象，其产生的政治内力足以瓦解独裁政权。

国内民众已基本被颅内监禁并被维稳系统完全监控，人民被完全驯化并丧失反抗意识，只会一味顺从压迫，并以维护极权统治为荣，但是这个片轮回无数次的土地上是再正常不过的现象，希望有血性的中国人尚在！

正如曾经阔气的要复古，正在阔气的要保持现状，未曾阔气的要革新，大抵如此，大抵！-鲁迅

**Comment by Kamijoucen** [comment]
不同意，你有些二极管了

**Reply by oceanzhang860411** [reply]
矯枉必須過正，並無不可

**Comment by Dreaming-Pepper** [comment]
你这就本末倒置了，红色政权，你说他蛊惑农民，实际上是，那一刻农民选择了红色政权，在那个年代，选择红色政权是极为正确的选择，通过掠夺土豪劣绅的财富壮大，农民和红色政权利益都是一致的。
国家成立后后，必须以国家的角度审视朝鲜战争、大跃进、大清洗、文革被、计划生育、天安门广场武力镇压等事件，显然对于掌控国家的人而言是尝试，人民要么推翻，要么跟随，但那一刻人民怕了选择妥协。而非做现代人骂过去，这就历史虚无主义了。
1994年的分税制改革确实是卖地炒房的起点，但说白了就是当时中央和地方谈判的结果，显然是当时最佳的选择，但后面纯粹就是中央能力不行。台湾更不可能武力夺权了，至少现在子虚乌有，纯粹为了扣帽子而扣帽子。
说白了现在这帮人不去大搞特搞的闹断贷。就是这帮人习得的小布尔乔亚的软弱性和妥协性，毕竟还剩个裤衩。
难道一定要等到连裤衩都没了，才去反抗吗？这才是我最无语的地方

**Reply by haoqing-yan** [reply]
老哥的辩证法可以的，讲的很客观

**Comment by Cegazhos** [comment]
如果你要搞反抗，我肯定口头支持你。但你扯到台湾，那我只能说你是傻卵一个。
我也不满政府所作所为，可是你怎么会扯到台湾去的呢？
你们台湾的能不能动动脑子 什么年代了 还隔这虚空绑定。
共产党确实是狗屎啊，可你们台湾的罪孽就不提了？

**Comment by cccp057** [comment]
这就是典型的胡说八道了。照你这个描述，共产党政府就不应该存在，可是他存在了70年诶？这不是自相矛盾吗？
至少1949年的时候，共产党政府是整个中国大地上最进步的政府。当时毛泽东讨论历史周期论说的很好，共产党可以突破历史周期律，这个法宝就是民主。这句话可以反过来理解，如果共产党不能做到真正的民主，那么就躲不了历史周期律。这就是毛主席革命的地方，也是很多中国人举他的画像说他的金句的原因。他的思想不仅可以对外颠覆，也可以用作对内革命。这是很多领袖所不具备的特质

**Reply by Cegazhos** [reply]
历史周期律早已经被现代科技所破坏了。寻常百姓不具备起义的条件。有梁山泊也活不过几天。只有掌控军队的才可能起义。这也是二战以来各个大国保持稳定的一个原因。当然，还有另一种形式 那就是苏联那种内部的崩溃。
这里回到毛说的法宝 民主，私以为这已经不适合新时代了。私以为他说的民主现在是共产党内部的民主，是政治局委员会里的民主。早就不是人大代表的那个民主了，最多也就是新法草案出来向社会征集意见的那种民主。但显然 这些是远远不够满足人们的表达欲的。
一个很明显的例子就是雷劈不死毛泽东的故事 不会再上演了。这是我不认可中修是红色的根本原因。

至于楼主[Kasianian](https://github.com/Kasianian)这般脑子僵硬只会将长时间的事物看成一个整体的想法，则无须讨论，没有价值。

**Reply by DancerCoding** [reply]
> 这就是典型的胡说八道了。照你这个描述，共产党政府就不应该存在，可是他存在了70年诶？这不是自相矛盾吗？ 至少1949年的时候，共产党政府是整个中国大地上最进步的政府。当时毛泽东讨论历史周期论说的很好，共产党可以突破历史周期律，这个法宝就是民主。这句话可以反过来理解，如果共产党不能做到真正的民主，那么就躲不了历史周期律。这就是毛主席革命的地方，也是很多中国人举他的画像说他的金句的原因。他的思想不仅可以对外颠覆，也可以用作对内革命。这是很多领袖所不具备的特质

马列主义的理论本身就脱离社会实际运行逻辑，通过洗脑传销渲染一个乌托邦社会，让那些目不识丁的农民或者粉红趋之若鹜或者当成燃料，再通过暴力革命夺取政权，建立自诩民主实际独裁的政权，共产党确实就不应该存在，或者可以说当时的那种环境，把马列主义当成传销工具造反而已。
毛泽东思想不过是马列主义思想的延申，是苏联异化思想的中国修正主义。我相信没有哪个政权愿意打脸不出修正主义维持自身合法性或者建立崇拜。
共产党自擂自夸，不把自己说成民族救星怎么对你们这些粉红实行反人类统治，请不要直接跳过共产党所犯下的反人类罪行。
说到历史周期律，在中国这片土地没有哪个政权可以跳出历史周期律，建议多读点鲁迅对中国人劣根性的描述，除非可以从根源处给中国人做手术，否则永远没有出头之路，只是不断地轮回罢了。
你现在还捧着毛泽东思想只能说被苏联余孽思想彻底洗脑了。包括现在的热武器时代，任何社会运行都有破解推倒的方法，只是奴隶没有血性只有顺从或者直接说不敢反抗，还有说历史虚无主义的人，难道我们祖先所承受的苦难不值得一提不值得吸取教训，就算我现在把你杀了，你儿子不能杀我是同一个道理。

**Comment by xiejun946** [comment]
台湾军事武装是中华民族内唯一制衡红色极权的有效力量，如果共产党胆敢武力夺取台湾，以独裁吞并民主的社会，是人类文明的倒退，共产党惧怕的杀戮形象会再次曝光在全世界的注目下，国际社会集体军事和经济制裁中国，加速禁锢并缩短独裁政权政治生命，即使共产党赢得吞并台湾的战争，也无法有效对台湾人民进行长期有效的统治，属于民主社会和红色极权的互斥现象，其产生的政治内力足以瓦解独裁政权。


看出来了，你就是个鸟台独分子。中国的事情，关你们台湾省什么事情。今天越过台湾海峡，金门关，明天军舰就开到你们台东，打死你个跷跷板

**Comment by a6749920** [comment]
只能说很多时候一个人都无能为力，环境如此 ，实现改革确实很难

**Comment by dofindme** [comment]
愚民政策，本就不好管理，没有精细化，所以一筐子打包归类！

**Comment by Ventery** [comment]
差不多得了😅
多少年了都是这几句，真是狗嘴里吐不出象牙来

**Reply by yuyoooo** [reply]
可不咋滴，半点长进没有

**Comment by wangsir003** [comment]
纵观历史，现在的中国人民活的最有尊严了吧，问了老一辈和爷爷姥爷辈都说现在生活更幸福，这怎么解释

**Reply by DancerCoding** [reply]
如果你问朝鲜人民生活幸福吗？朝鲜人民肯定饥寒交迫并含着泪说：“有金正恩的伟大领袖的坚强领导，我们生活一定会很幸福”
如果你问日本人幸福吗？日本人会鄙笑着说：“这人有病”
那么请问到底谁生活更幸福？所以经济基础决定上层建筑本身就是悖论，没有平权自由的制度哪来更幸福的人民？

奴隶整天喜欢把人民靠自己劳动创造的成果成是独裁政权的功劳，可实际独裁政权的权贵们只会趴在人民头上吸血，然後把政权所犯下的反人类罪行抛掷脑後，等待下一轮苦难的到来，然後自诩制度的伟大。

所以你永远叫不醒一个装睡还自我感动的奴隶。

**Reply by wedxz** [reply]
> 纵观历史，现在的中国人民活的最有尊严了吧，问了老一辈和爷爷姥爷辈都说现在生活更幸福，这怎么解释

有个几把尊严了？哪个朝代有过把两个家庭三代人拼命凑的首付钱结果买到了烂尾楼，还不让停贷？

**Reply by xiejun946** [reply]
> 如果你问朝鲜人民生活幸福吗？朝鲜人民肯定饥寒交迫并含着泪说：“有金正恩的伟大领袖的坚强领导，我们生活一定会很幸福” 如果你问日本人幸福吗？日本人会鄙笑着说：“这人有病” 那么请问到底谁生活更幸福？所以经济基础决定上层建筑本身就是悖论，没有平权自由的制度哪来更幸福的人民？
> 
> 奴隶整天喜欢把人民靠自己劳动创造的成果成是独裁政权的功劳，可实际独裁政权的权贵们只会趴在人民头上吸血，然後把政权所犯下的反人类罪行抛掷脑後，等待下一轮苦难的到来，然後自诩制度的伟大。
> 
> 所以你永远叫不醒一个装睡还自我感动的奴隶。

网上这么叫嚣又作出什么行动吗？出书还是变革，也没见你在大陆墙内传法，不过叫嚣士罢了，说的这么多，一点行动都没有

**Reply by DancerCoding** [reply]
> > 如果你问朝鲜人民生活幸福吗？朝鲜人民肯定饥寒交迫并含着泪说：“有金正恩的伟大领袖的坚强领导，我们生活一定会很幸福” 如果你问日本人幸福吗？日本人会鄙笑着说：“这人有病” 那么请问到底谁生活更幸福？所以经济基础决定上层建筑本身就是悖论，没有平权自由的制度哪来更幸福的人民？
> > 奴隶整天喜欢把人民靠自己劳动创造的成果成是独裁政权的功劳，可实际独裁政权的权贵们只会趴在人民头上吸血，然後把政权所犯下的反人类罪行抛掷脑後，等待下一轮苦难的到来，然後自诩制度的伟大。
> > 所以你永远叫不醒一个装睡还自我感动的奴隶。
> 
> 网上这么叫嚣又作出什么行动吗？出书还是变革，也没见你在大陆墙内传法，不过叫嚣士罢了，说的这么多，一点行动都没有

你一个非暴力不合作都不会、兼没血性奴才，就算我给你传法你敢做吗？真到那时候你不会在旁边看戏吧?你还是下楼做核酸吧，扫扫场所码，尽早上贵阳转运的那辆大巴比较实际

**Reply by xiejun946** [reply]
咬人的狗不叫，不是1450，就是个崇洋媚外的家伙

**Reply by kuankuanlv** [reply]
> 纵观历史，现在的中国人民活的最有尊严了吧，问了老一辈和爷爷姥爷辈都说现在生活更幸福，这怎么解释

你怎么不和元谋人、周口人比呢？

**Reply by maxwillma** [reply]
> 纵观历史，现在的中国人民活的最有尊严了吧，问了老一辈和爷爷姥爷辈都说现在生活更幸福，这怎么解释

周公子和它爷爷们 肯定是幸福的，至于 一个月100 RMB 退休金的老农民，那就不一定了。

**Comment by b4o** [comment]
LZ就是SB，国民党统治的时候是什么样子，你脑子瘫眼睛瞎，你在全世界找一个近几十年经济，生活，国力发展速度能比上中国的再说吧，SB一个，你要认爸爸就去，没人阻拦你，别天天叫唤

**Reply by JayChou55** [reply]
全球有几个比中国人勤劳。。。新加坡都超香港了 你怎么说

**Reply by anlondon** [reply]
> 全球有几个比中国人勤劳。。。新加坡都超香港了 你怎么说

香港就是个中转买办的地，专门为内陆采买国外物资技术的窗口以赚取中间费。现在内陆发展起来了，自己就可以去买。香港还是老一套自然地位下来了。

来自wiki:
> 香港经济是高度依赖国际贸易的自由市场经济系统，作为全球服务业主导程度最高的经济体，金融服务业（2019年该业增加值占GDP的21.2%）、贸易及物流业（19.8%）、专业服务及其他工商业支援服务（11.9%）、旅游业（3.6%）是香港经济的四个传统主要行业。
> 
> 新加坡为全球第三大、亚洲最大的国际金融中心，也拥有以电子、能源化工、精密工程为代表的先进制造业，为全球第四大高科技产品出口国。

另外香港就是个省，新加坡是国家。你应该这么问，中国人这么勤劳，中国什么时候超过新加坡🤪？

**Reply by lance111l** [reply]
新加坡只是小小国家   如果成为世界第二  美国肯定弄死他

**Comment by lance111l** [comment]
收复台湾是20年内的事   中国的发展速度可预期 
台湾太小了  如果搞台独  马上会被痛打   经济一落千丈  人民水深火热   不用多久就被灭了

**Comment by Corivesper1021** [comment]
> 一说到这个肯定有苏联粉红余孽跳出来说你是1450和行走的50w，但是作为正统中国人，就是要有一份光发一份热，不必等候矩火。
> 
> 1，红色极权的发家史无非就是利用苏联共产主义传销思想，蛊惑无数农民为不可能实现的目标充当燃料并燃烧自身生命，推翻了所谓旧社会，迎来新的一轮更甚的地狱模式，其社会制度与封建制度并无差异，俗称资本主义中最低端的官僚资本主义。
> 
> 2，红色极权手上沾满了无数无辜百姓被迫害的鲜血，本不应该发生的国共内战、朝鲜战争（造成朝鲜人民彻底生活在奴隶社会的悲剧）、三年饥荒、大跃进、毛泽东的斯大林模式的大清洗运动、文革被迫害致死的上千万知识分子、计划生育阉割屠杀上亿汉族婴儿，以及1989年天安门广场武力镇压屠杀学运学生等等，目前政权仍然不公开自省甚至对人民封口管制。
> 
> 3，自计划经济严重限制社会生产力发展，为延续政治生命被迫推行改革开放，其四十年发展成果无非是建立在燃烧无数廉价劳动力基础上而发展出的畸形经济体制，国有垄断生产资料企业=红色权贵的官僚门阀企业、共产党垄断土地造成土地归属权混乱，有进一步被共产党红色干部进行土地兼并的情况，独裁体制下无限扩大的权力无法被有效监督和制衡，无法根治体制内生性的大范围红色权贵贪腐问题。
> 
> 4，支持极权续命的税收制度已无法完成闭环，1994年的分税制改革，中央财政直接对地方财政抽成25%，造成地方财政严重依赖卖地收入，造成经济发展严重依赖房地产，每年卖地收入被用来红色权贵裙带关系户挥霍，进行的所谓大基建以供其吸血，目前大部分大基建项目都是政府的资产负债项，其中港珠澳大桥就是其缩影，往後地方政府只能靠发债维持庞大的党政体制度日，没有美帝的实力却裹上了美帝的赤字，直至土崩瓦解。
> 
> 5，疫情的禁足防疫已无法使经济进行有效内循环和制造业出口，外资看空禁足防疫，大范围企业倒闭，大规模失业潮来临，同时一带一路还给外国大爷撒钱，援非项目不但撒钱还提供免费劳动力。
> 
> 6，台湾军事武装是中华民族内唯一制衡红色极权的有效力量，如果共产党胆敢武力夺取台湾，以独裁吞并民主的社会，是人类文明的倒退，共产党惧怕的杀戮形象会再次曝光在全世界的注目下，国际社会集体军事和经济制裁中国，加速禁锢并缩短独裁政权政治生命，即使共产党赢得吞并台湾的战争，也无法有效对台湾人民进行长期有效的统治，属于民主社会和红色极权的互斥现象，其产生的政治内力足以瓦解独裁政权。
> 
> 国内民众已基本被颅内监禁并被维稳系统完全监控，人民被完全驯化并丧失反抗意识，只会一味顺从压迫，并以维护极权统治为荣，但是这个片轮回无数次的土地上是再正常不过的现象，希望有血性的中国人尚在！
> 
> 正如曾经阔气的要复古，正在阔气的要保持现状，未曾阔气的要革新，大抵如此，大抵！-鲁迅

国内有问题是肯定的，这需要社会主义自我改革，不断完善。我们初期不走剥削农民哪来的原始积累（苏联也是），哪来的后面的工业化？我只能说多亏了毛主席带领人民走了出来，为了整个民族的大局，才让我们过的上一天三外卖的好日子。虽然国家与个人的关系并不是绝对的，但肯定整体而言国家好个人才好（参考当今乌克兰，那群妹子都干啥去了你猜？）。欧美他们怎么发家的你装傻呢？考对外殖民扩张，我们有实力吗？没有，好不容易赶走殖民者你还欢迎人家回来？建国之前国内烂成什么样子了你不做调研？奉劝一句，还有良心的话那就少诋毁中国，除非你一家子早就是汉奸走狗；如果你是外国人来搞舆论的，那对不起，中国走什么道路你管不着，而且我就喜欢看你们气急败坏的样子，等到你们先发国家红利吃完，看你们还怎么得瑟嘿嘿

**Reply by Cegazhos** [reply]
多读历史。至少读中修自己的党史 材料再出来侃侃而谈。否则一句话n个历史错误 别人都懒得搭理你。

**Reply by maxwillma** [reply]
这就是 教育 这个公器，被私用的后果

**Comment by lance111l** [comment]
收回台湾是中国人的历史使命  没有余地    哪怕摧毁一切阻力

**Comment by Keyuandxie** [comment]
一群负能量的人 见不得中国好 你们能咋地 中国共产党至少还是上升期 汇集了不少民心民意  你们什么也阻止不了 只能在历史的大潮湮灭 现在大多数人都认可共产党统治

**Comment by b4o** [comment]
只能说楼主对中国太不了解了，不知道你是CAI还是华独，红色政权从26个人发展到600万军队，难道那些军队是天上掉下来的？每一个战士都是你口中说的农民，没有红色政权之前中国是半殖民半封建社会，大上海还有”狗和中国人不得入内“，有了红色政权外国人还有原来那么猖狂吗？
奉劝楼主，不甘心就去好好磨刀直接干，搞舆论？就想把中国搞烂，你太天真了，你能让中国14人全部有饭吃？能让6亿人脱贫？能让家家户户通电通路？不要把所有中国人当成傻子，你能洗脑的只是极少数极少数，而且是那些被看不起的牧羊犬，就算他们全都上街游行，对中国14亿人来说都是极小数，中国政府就是要保护80%人的利益，不可能让所有人满意，请问你喜欢的政府政党在自己的国家让多少人满意了？

**Reply by maxwillma** [reply]
义务教育学的真好

**Comment by maomaoc474** [comment]
> 一说到这个肯定有苏联粉红余孽跳出来说你是1450和行走的50w，但是作为正统中国人，就是要有一份光发一份热，不必等候矩火。
> 
> 1，红色极权的发家史无非就是利用苏联共产主义传销思想，蛊惑无数农民为不可能实现的目标充当燃料并燃烧自身生命，推翻了所谓旧社会，迎来新的一轮更甚的地狱模式，其社会制度与封建制度并无差异，俗称资本主义中最低端的官僚资本主义。
> 
> 2，红色极权手上沾满了无数无辜百姓被迫害的鲜血，本不应该发生的国共内战、朝鲜战争（造成朝鲜人民彻底生活在奴隶社会的悲剧）、三年饥荒、大跃进、毛泽东的斯大林模式的大清洗运动、文革被迫害致死的上千万知识分子、计划生育阉割屠杀上亿汉族婴儿，以及1989年天安门广场武力镇压屠杀学运学生等等，目前政权仍然不公开自省甚至对人民封口管制。
> 
> 3，自计划经济严重限制社会生产力发展，为延续政治生命被迫推行改革开放，其四十年发展成果无非是建立在燃烧无数廉价劳动力基础上而发展出的畸形经济体制，国有垄断生产资料企业=红色权贵的官僚门阀企业、共产党垄断土地造成土地归属权混乱，有进一步被共产党红色干部进行土地兼并的情况，独裁体制下无限扩大的权力无法被有效监督和制衡，无法根治体制内生性的大范围红色权贵贪腐问题。
> 
> 4，支持极权续命的税收制度已无法完成闭环，1994年的分税制改革，中央财政直接对地方财政抽成25%，造成地方财政严重依赖卖地收入，造成经济发展严重依赖房地产，每年卖地收入被用来红色权贵裙带关系户挥霍，进行的所谓大基建以供其吸血，目前大部分大基建项目都是政府的资产负债项，其中港珠澳大桥就是其缩影，往後地方政府只能靠发债维持庞大的党政体制度日，没有美帝的实力却裹上了美帝的赤字，直至土崩瓦解。
> 
> 5，疫情的禁足防疫已无法使经济进行有效内循环和制造业出口，外资看空禁足防疫，大范围企业倒闭，大规模失业潮来临，同时一带一路还给外国大爷撒钱，援非项目不但撒钱还提供免费劳动力。
> 
> 6，台湾军事武装是中华民族内唯一制衡红色极权的有效力量，如果共产党胆敢武力夺取台湾，以独裁吞并民主的社会，是人类文明的倒退，共产党惧怕的杀戮形象会再次曝光在全世界的注目下，国际社会集体军事和经济制裁中国，加速禁锢并缩短独裁政权政治生命，即使共产党赢得吞并台湾的战争，也无法有效对台湾人民进行长期有效的统治，属于民主社会和红色极权的互斥现象，其产生的政治内力足以瓦解独裁政权。
> 
> 国内民众已基本被颅内监禁并被维稳系统完全监控，人民被完全驯化并丧失反抗意识，只会一味顺从压迫，并以维护极权统治为荣，但是这个片轮回无数次的土地上是再正常不过的现象，希望有血性的中国人尚在！
> 
> 正如曾经阔气的要复古，正在阔气的要保持现状，未曾阔气的要革新，大抵如此，大抵！-鲁迅

傻叉啊！你是不是不知道 中国台湾 蒋秃驴 是世界上最长的军统政府 实施了全世界最久的独裁宵禁， 仅仅次于今天的叙利亚政府。制造了震惊世界的白色恐怖，可以说是中华民族的耻辱。 和大陆比起来，台湾的近代史就是一泡屎。无论民心，民主还是实力， 谁给你这个傻逼的权利来评论大陆政府的，傻逼台蛙有资格吗？祖宗的脸都被丢干净了，丢了政权，丢了领土，抱洋人大腿的畜生政府！

**Comment by liuyifan111223** [comment]
说白了，这样极品的自由派头一次见。所以我有个问题，中共要是真不行，共产主义意识形态不行，那冷战是干什么的？即使现在大陆有些许偏差但是意识形态总路线是不可能变的，资产阶级政府搁中国活不长，这个破命题自毛泽东到现在论证了无数遍了，现在撑死算国家资本主义，你跟我扯官僚？中国大陆政治部门，军事部门，经济，党政部门，人民代表大会是分开的。自毛泽东主席逝世的时候就一直是这样的，撑死人大与党内直接干预，政府压根做不到直接支配经济。贪官可不是商人。这种言论明显错误至极
